Pearson-Advance/frontend-app-institution-portal
# PADV-560 - Create base repository using MFE template.

## Ticket

https://agile-jira.pearson.com/browse/PADV-560

## Description

In order to meet the new business requirements for institution admins, it is essential to develop a new portal to allow access to Institution user. A new MFE implementation is required for this portal, so we need to use the frontend-template:  https://github.com/openedx/frontend-template-application project to create the base structure for this Institution Portal MFE.

![image](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/30726391/b43f6df5-f26b-4d21-affe-27e04b4cb3d1)


## Changes made

- [x] Forked repository from openedx MFE template. The new repository must be called: frontend-app-institution-portal.
- [x] Remove unnecessary definitions and files.
- [x] Github actions enabled.
- [x] Unnecessary Github Actions have been removed.

## How to test

- Clone the Institution Portal MFE
- Run openedx services
- Run npm install.
- Run npm run lint
- Run npm test
- Set `"RUNTIME_PROVE_SETTING": "EXAMPLE"` MFE_CONFIG in site_configurations.
- Run npm run start
- Go to http://http://localhost:1627 and you should see the value of `RUNTIME_PROVE_SETTING`
